### DISCUSSION TITLE
Add `mass` sensor class

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#808] [Entity Models] [answered]
## Proposal

Add a `sensor` `device_class` for `mass` to home assistant core and frontend (+ documentation).

To solve the problem with different applications using different units for mass, without the need to add device classes for each of those applications, the unit of a `mass` sensor shall be configurable by the user in the same way as for temperature and pressure sensors.

Specific device classes for common applications may of course be added if there is a need.

A draft PR can be found in https://github.com/home-assistant/core/pull/78993

**Comment by emontnemery** [comment, marked as answer]
We've discussed this topic and concluded that we should add a `weight` sensor device class instead, as proposed by https://github.com/home-assistant/architecture/discussions/498.
